source/common/grpc/async_client_impl.cc

```diff
@@ -190,33 +223,37 @@ void AsyncStreamImpl::initialize(bool buffer_body_for_retry) {
 void AsyncStreamImpl::onHeaders(Http::ResponseHeaderMapPtr&& headers, bool end_stream) {
```

Comment:
Should we remove this TODO comments?

Reply:
per recent discussion, we want to drop the base64 decoding of server side -bin metadata until https://github.com/envoyproxy/envoy/issues/39054 resolves.